### PR TITLE
Feature/test board

### PR DESCRIPTION
**Pull request by hervecarol**
**Warning :** Before creating a pull request, make sure that it does not already exists in the [pull requests](../). Thank you.

## Description
Added the test board module

## Fixes
Link all the related issues from the issue tracker
- Closes: #

## How has this been tested ?
(If possible) describe the steps to test these modifications.

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/src/components/TestBoardModule.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    const [rate, setRate] = React.useState("")
+    const [isSingleSend, setIsSingleSend] = React.useState(true)
+    let intervalVar: any
+    const testBoardPublisher = useROSTopicPublisher<any>("/interface_rs485/dataRx", "sonia_common/SendRS485Msg")
+
+    const handleSubmit = () => {
+        console.log(slave)
+    }
+    const handleStart = () => {
+        console.log("Slave: ", slave)
```
Comment (antho6222) [inline src/src/components/TestBoardModule.tsx:32]: console log oublié? :)

File: src/src/components/TestBoard.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React, { useCallback } from 'react';
+
+const TestBoard = () => {
+
```
Comment (antho6222) [inline src/src/components/TestBoard.tsx:4]: A quoi sert ce fichier? il semble inutile (oubli de retirer?)

File: src/src/components/TestBoardModule.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    const [slave, setSlave] = React.useState("")
+    const [cmd, setCmd] = React.useState("")
+    const [data, setData] = React.useState("")
+    const [rate, setRate] = React.useState("")
+    const [isSingleSend, setIsSingleSend] = React.useState(true)
+    let intervalVar: any
+    const testBoardPublisher = useROSTopicPublisher<any>("/interface_rs485/dataRx", "sonia_common/SendRS485Msg")
+
+    const handleSubmit = () => {
+        console.log(slave)
```
Comment (antho6222) [inline src/src/components/TestBoardModule.tsx:29]: On fait rien sur le submit?

**Review by antho6222 — CHANGES_REQUESTED** [review]

File: src/src/components/TestBoardModule.tsx (hunk 3 above)
Comment (hervecarol) [reply src/src/components/TestBoardModule.tsx]: Non on fait rien parce que tout est géré par les boutons start et stop. La methodle handleSubmit va etre enlevé

**Pull request merged**